src/modules/DialogOverlay.vue

```diff
@@ -72,12 +74,25 @@ export default defineComponent({
     toggleDialog() {
       this.$emit('toggle-dialog')
     },
-    getPrimaryProductData(data: ProductItemProps[]): TabItemProps[] {
-      return data[0].productItems
+    selectProductType(): ProductEnum {
+      const productType = this.tabsData[0].productType
+      return productType
+    },
```

Comment:
for the time being, we are waiting on the designers to tell us how this selection will be done. Till then, we will only show the first product returned from BE